spec/unit/util/log_spec.rb

```diff
@@ -18,6 +18,32 @@ def log_notice(message)
     expect(message).to eq("foo")
   end
 
+  context "given a message with invalid encoding" do
+    let(:logs) { [] }
+    let(:invalid_message) { "\xFDfoo".force_encoding(Encoding::UTF_8) }
```

Comment:
Maybe better to test with an invalid byte of another multi-byte encoding, like Shift-JIS?

```
[4] pry(main)> "\xFD\xFB".force_encoding(Encoding::Shift_JIS).valid_encoding?
=> false
```